narwhal/primary/tests/nodes_bootstrapping_tests.rs

```diff
@@ -134,18 +134,15 @@ async fn test_second_node_restart() {
 #[tokio::test]
 /// We are testing the loss of liveness of a healthy cluster. While 3f+1 nodes run
 /// we are shutting down f+1 nodes. Then we are bringing the f+1 nodes back again
-/// but we are not expecting now the cluster to be able to make progress. We expect
-/// the restarted nodes to not be able to make new proposals and effectively make
-/// the system stall. This is due to the issue that is described here
-/// https://github.com/MystenLabs/narwhal/issues/664 .
-/// Once this is fixed we would expect this test to fail as all the nodes should be
-/// able to propose from where they left of at last round.
+/// We expect the restarted nodes to be able to make new proposals, and all the nodes
+/// should be able to propose from where they left of at last round, and the rounds should
+/// all advance.
 async fn test_loss_of_liveness_without_recovery() {
     // Enabled debug tracing so we can easily observe the
     // nodes logs.
     let _guard = setup_tracing();
 
-    let node_advance_delay = Duration::from_secs(60);
+    let node_advance_delay = Duration::from_secs(10);
```

Comment:
I would recommend to leave this back to `60` - we do want to give our tests the room to advance and reveal any possible pathogens which might not be visible in short time